packages/webostv/template/src/index.js

```diff
@@ -0,0 +1,31 @@
+import {error} from '@enact/webos/pmloglib'
+import React from 'react';
+import {render} from 'react-dom';
+
+import App from './App';
+
+const appElement = (<App />);
+
+// In a browser environment, render instead of exporting
+if (typeof window !== 'undefined') {
+	window.addEventListener('error', function (ev) {
```

Comment:
Unfortunately, can't add a listener at modular level, or it will not get correctly added to the window in v8 snapshot scenario.  Will probably need to be moved to within the App lifecycle functions.

Depending whether we expect the apps to want to customize the error handling (and whether its considered an optional feature for app devs), we may want to consider consolidating the error handling support into a module/function within `@enact/webos`, then invoking that within the template instead. Based off discussions, we'd rather avoid tying `@enact/moonstone` directly to `@enact/webos` so having the template be the entity to access `@enact/webos` seems like an acceptable alternatively (in the event we go that route).